modules/custom/d_p_carousel/templates/paragraph--d-p-carousel.html.twig

```diff
@@ -56,11 +55,14 @@ view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
         <div class="content-wrapper">
           <div class="content container">
             {% block content %}
-              {{ content }}
+              {{ content|without('field_d_p_cs_item_reference') }}
             {% endblock %}
           </div>
+          <div class="full-width-content">
```

Comment:
A jeśli nie będzie klasy `.full-width` to czy karuzela nadal będzie rozciągnięta od krawędzi do krawędzi ekranu?
Bo w takim przypadku powinna być w kontenerze. A wygląda na to, że `.full-width-content` załapie tu zawsze.